examples/join-tables-and-streams/join-tables-and-streams.description

```diff
@@ -0,0 +1,5 @@
+//Tables and steams can be joined using streaming query logic.
+//In this example, orders coming from customers for a certain item are checked against the stock table to verify
+//whether there is enough stock to deliver the order. An alert is triggered if the stock is not sufficient.
+//Ballerina Streams is an experimental feature. Please use `--experimental` flag when compiling Ballerina files
+//that have streaming constructs.
```

Comment:
Shall we add a new line at the end of file?

Reply:
done